pt-BR/i18n.md

```diff
@@ -84,67 +84,66 @@ backend                   # Use um backend diferente
 
-The **translations load path** (`I18n.load_path`) is an array of paths to files that will be loaded automatically. Configuring this path allows for customization of translations directory structure and file naming scheme.
+O **caminho de carregamento de traduções* (`I18n.load_path`) é um *array* de caminhos para arquivos que serão carregados automaticamente. Configurar este caminho permite a personalização da estrutura do diretório de traduções e do esquema de nomeação de arquivos.
 
-NOTE: The backend lazy-loads these translations when a translation is looked up for the first time. This backend can be swapped with something else even after translations have already been announced.
+NOTE: O *backend* carrega essas traduções de forma lenta quando uma tradução é consultada pela primeira vez. Esse *backend* pode ser trocado por outro mesmo depois que as traduções já foram anunciadas.
 
-You can change the default locale as well as configure the translations load paths in `config/application.rb` as follows:
+Você pode alterar a localidade padrão, bem como configurar os caminhos de carregamento de traduções em `config/application.rb` da seguinte forma:
 
 ```ruby
 config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}')]
 config.i18n.default_locale = :de
 ```
-
-The load path must be specified before any translations are looked up. To change the default locale from an initializer instead of `config/application.rb`:
+O caminho de carregamento deve ser especificado antes de qualquer consulta de tradução. Para alterar a localidade padrão a partir de um inicializador em vez de `config/application.rb`:
 
 ```ruby
 # config/initializers/locale.rb
 
-# Where the I18n library should search for translation files
+# Onde a biblioteca I18n deve procurar arquivos de tradução
 I18n.load_path += Dir[Rails.root.join('lib', 'locale', '*.{rb,yml}')]
 
-# Permitted locales available for the application
+# Localidades permitidas disponíveis para a aplicação
 I18n.available_locales = [:en, :pt]
 
-# Set default locale to something other than :en
+# Definir localidade padrão para algo diferente de :en
 I18n.default_locale = :pt
 ```
 
-Note that appending directly to `I18n.load_path` instead of to the application's configured i18n will _not_ override translations from external gems.
+Note que anexar diretamente a `I18n.load_path` em vez de à configuração i18n da aplicação _não_ substituirá traduções de gems externas.
```

Comment:
```suggestion
Note que anexar diretamente a `I18n.load_path` em vez de à configuração i18n da aplicação **não** substituirá traduções de _gems_ externas.
```

Uma sugestão meio tosca mas vendo talvez faça sentido, veja o que acha.

Como usamos bastante itálico pros termos extrangeiros talvez possamos trocar esse por negrito